src/config/index.js

```diff
@@ -36,6 +36,7 @@ const configuration = {
   LOGO_TRADEMARK_URL: process.env.LOGO_TRADEMARK_URL,
   USE_API_CACHE: process.env.USE_API_CACHE,
   PLOTLY_SERVER_URL: process.env.PLOTLY_SERVER_URL,
+  ENTERPRISE_SUPPORT_LEARNER_CREDIT_URL: process.env.ENTERPRISE_SUPPORT_LEARNER_CREDIT_URL,
```

Comment:
This configuration file should be considered legacy at this point; instead, I would recommend adding the new config setting into the `AppContext` provided by `@edx/frontend-platform` so that the config setting is available via `getConfig` from frontend-platform.

This can be done by adding this environment variable to the `mergeConfig` call in `src/index.jsx` ([source](https://github.com/openedx/frontend-app-admin-portal/blob/9b6f100ec9b3279436ee28ffbe1fb375ed0f75ac/src/index.jsx#L31-L42)).